### PR TITLE
Add happy hours block

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: idunn/blocks/happy_hour.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from humanized_opening_hours.exceptions import HOHError
+
+from .base import BaseBlock
+from .opening_hour import get_tz, get_coord, tz, parse_time_block, get_days
+
+
+logger = logging.getLogger(__name__)
+
+
+class HappyHoursType(types.Type):
```
Comment (amatissart) [inline idunn/blocks/happy_hour.py]: why not using the same class as in opening_hours ?

File: idunn/places/pj_poi.py [hunk 2]
```diff
@@ -56,7 +56,13 @@ def get_subclass_name(self):
         return self.get_class_name()
 
     def get_raw_opening_hours(self):
-        opening_hours_dict = self.get('OpeningHours', {})
+        return self.get_raw_hours_kind('OpeningHours')
+
+    def get_raw_happy_hours(self):
+        return self.get_raw_hours_kind('HappyHours')
```
Comment (amatissart) [inline idunn/places/pj_poi.py]: This field is not defined in the source data, we could probably ignore happy hours in this class.

**Comment by GuillaumeGomez** [comment]
Updatd.

File: idunn/places/pj_poi.py [hunk 3]
```diff
@@ -56,7 +56,10 @@ def get_subclass_name(self):
         return self.get_class_name()
 
     def get_raw_opening_hours(self):
-        opening_hours_dict = self.get('OpeningHours', {})
+        return self.get_raw_hours_kind('OpeningHours')
+
+    def get_raw_hours_kind(self, kind):
+        opening_hours_dict = self.get(kind, {})
```
Comment (amatissart) [inline idunn/places/pj_poi.py]: changes in this file are no longer needed

File: idunn/blocks/__init__.py [hunk 4]
```diff
@@ -1,16 +1,32 @@
...
 from .grades import GradesBlock
-from .events import OpeningDayEvent, DescriptionEvent
+from .events import (
+    OpeningDayEvent,
+    DescriptionEvent,
+)
 
 ALL_BLOCKS = [
     OpeningHourBlock,
+    HappyHourBlock,
```
Comment (amatissart) [inline idunn/blocks/__init__.py]: this block also needs to be defined in `BLOCKS_BY_VERBOSITY` object (in utils.py), to be included in the "long" verbosity output.

File: idunn/places/pj_poi.py [hunk 5]
```diff
@@ -57,7 +57,6 @@ def get_subclass_name(self):
 
     def get_raw_opening_hours(self):
         opening_hours_dict = self.get('OpeningHours', {})
-        raw = ""
```
Comment (amatissart) [inline idunn/places/pj_poi.py]: this breaks some tests

**Pull request merged**